### ISSUE TITLE
Reverse normalize on batches of images?(tensor type not PIL image)

### ISSUE THREAD

**Issue by Charlie4zc** [pytorch/pytorch#22482]
I have batches of images with size of (B,C,H,W) (torch.tensor) i want to unnormalized it with mean/std tensor (e.g.:[0.4,0.3,0.2]) with size of(C,H,W)

My question is is there any operations in Pytorch can do this like dataset.map or tf.expand_dims in Tensorflow ?

Be careful ,it not PIL image format
